service/lib/dinstaller/software.rb

```diff
@@ -59,14 +61,16 @@ def probe(progress)
       Yast::Pkg.SourceRestore
       Yast::Pkg.SourceLoad
       progress.next_minor_step("Making initial proposal")
-      @products = Y2Packager::Product.available_base_products
+      @products = Y2Packager::Product.available_base_products.select do |product|
+        SUPPORTED_PRODUCTS.include?(product.name)
+      end
```

Comment:
in general looks good, but please add more logging, so we know what is going on.